### DISCUSSION TITLE
Gestion des observations (scientifiques)

### DISCUSSION THREAD

**Discussion by urien** [GrottoCenter/grottocenter-front#877] [Ideas]
Un travail a été déjà amorcé mais n'est pas encore mis en place. 

Des issues côté front et coté back concernent la solution qui a été retenue et qui est décrite ici

- Toutes les observations sont accessibles via une URL de type ui/observations/id
- Les observations peuvent être recherchée via la recherche avancée, elles sont associées à une catégorie d'observation
- Les observations sont pour l'essentiel associées à une cavité. Mais certaines fonctionnalités devront permettre de prendre en compte plusieurs entrées, éventuellement des entrées et des points. Par exemple la gestion des traçages (écoulement d'eau) ou la visualisation de données issues de capteurs
- Il est envisageable d'associer une observation à un point qui est indépendant d'une entrée (par exemple un point où sont mesurées les précipitation ou un point où des mesures sont réalisées). Il faudra être mesure ultérieurement d'associer l'observation avec un point dans la cavité.
- une seconde page sera créée pour toutes les entrées qui permettra d'afficher les données de base ainsi qu'un résumé de toutes les observations attachées à la cavité. 
- les observations pourraient être collectées grâce à des outils mis en place sur Grottocenter ou le service pourrait être pris en charge par une structure tierce. Les critères pour que soit envisagé de s'appuyer sur un service tiers pourraient être ce qui est défini par Karstlink http://uisic.uis-speleo.org/wiki/karstlink/index.php/Karstlink_Applications_-_Applications_Karstlink

Les observations à collecter seraient
- Les données sur les activités humaines dans les cavités
- Les données issues de capteurs
- Les données biospéléologies
- Les données sur les pollutions du milieu karstique
- Les données sur la géologie
- Les données sur les concrétions
- Les topographies
- Les données liées à l'hydrologie
- Les données archéologiques

**Comment by Souterweb** [comment]
J'ajouterais bien :
- des données archéologiques.

Sachant que ce type de données sera probablement lié à des cavités à accès restreint il faudra, à terme, pouvoir définir qui aura les droits d'accès sur ce type de données et comment les gérer.

**Reply by urien** [reply]
ajouté

**Comment by BTHOM** [comment]
ajouter : les données liées à l'aérologie

**Reply by BTHOM** [reply]
éventuellement aussi : les données liées à la sismologie
